articles/energy-data-services/how-to-manage-data-security-and-encryption.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Microsoft Energy Data Services - Data Security and Encryption in Microsoft Energy Data Services #Required; page title is displayed in search results. Include the brand.
+description: Guide on security in Microsoft Energy Data Services and how to set up customer managed keys on Microsoft Energy Data Services #Required; article description that is displayed in search results. 
+author: sandeepchads #Required; your GitHub user alias, with correct capitalization.
+ms.author: sancha #Required; microsoft alias of author; optional team alias.
+ms.service: energy-data-services #Required; service per approved list. slug assigned by ACOM.
+ms.topic: how-to #Required; leave this attribute/value as-is.
+ms.date: 09/29/2022
+ms.custom: template-concept #Required; leave this attribute/value as-is.
+
+#Customer intent: As a developer, I want to set up customer-managed keys on Microsoft Energy Data Services
+---
+# Data Security and Encryption in Microsoft Energy Data Services
+
+This article provides an overview of security features in Microsoft Energy Data Services. It covers the major areas of [encryption at rest](../security/fundamentals/encryption-atrest.md), encryption in transit, TLS, https, microsoft-managed keys, and customer managed key. 
```

Comment:
Suggested change: HTTPS, Microsoft-managed keys (MMK), and Customer-managed Encryption Keys (CMEK)